predictor_pol/flask_app.py

```diff
@@ -98,52 +97,71 @@ def main():
                     )
 
         if validate(session):
-            answer_id = save_response(session)
+            predictions = predict(session)
+            answer_id = save_response(session, predictions['candidate_id'])
             session['answer_id'] = answer_id
-            return render_template('success.html')
+
+            return render_template('success.html',
+                                   predicted_candidate_name=predictions['candidate_name'],
+                                   candidatos = CANDIDATOS)
```

Comment:
```suggestion
                                   candidatos=CANDIDATOS)
```

Reply:
Also it would be more concise with the overall style if we did something like this.
```python
return render_template(
    'success.html',
    predicted_candidate_name=predictions['candidate_name'],
    candidatos=CANDIDATOS
)
```